cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -89,6 +89,7 @@
 
   // Some state for external service to query
   private final AtomicBoolean _stopRequested;
+  private final AtomicBoolean _forceStop;
```

Comment:
Rather than having a separate `AtomicBoolean` here, do you think we can mimic the way Unix kill signals work -- e.g. we can assign a signal id using a single `AtomicInteger`? This also helps us make the kill logic more extensible.

Please see: https://www.gnu.org/software/libc/manual/html_node/Termination-Signals.html